kytos/core/config.py

```diff
@@ -145,6 +145,10 @@ def parse_args(self):
 
         defaults.update(dict(config.items("daemon")))
 
+        # Gets vlan_pool after default is updated with values
+        # from config gile. then parses vlan_pool via json.loads
+        defaults.update(vlan_pool=json.loads(defaults.get('vlan_pool')))
```

Comment:
It seems that the most appropriate place for this code is the `_parse_options` method, just below this one.

Reply:
As the team and I were implementing this, we thought the same. We shall update the code and test again as soon as we can.